subprojects/core-api/src/main/java/org/gradle/api/artifacts/MinimalExternalModuleDependency.java

```diff
@@ -23,7 +23,7 @@
  * @since 6.8
  */
 @HasInternalProtocol
-public interface MinimalExternalModuleDependency {
+public interface MinimalExternalModuleDependency extends ExternalModuleDependency {
```

Comment:
I'd consider renaming one of these interfaces, this line reads weird, in the sense of: "Why is the minimal interface adding methods?"